tenbilac/committee.py

```diff
@@ -0,0 +1,189 @@
+"""
+This is Tenbilac!
+This is a net (Wnet in the future) ensemble class.
+"""
+
+import numpy as np
+from multiprocessing import Process, Pipe, Queue
+from itertools import izip  
+
+import logging
+logger = logging.getLogger(__name__)
+
+from . import train
+from . import utils
+
+
+def spawn(f):  
+	def fun(pipe,x):  
+		pipe.send(f(x))  
+		pipe.close()  
+	return fun  
+
+def parmap(f, X, ncpu):  
+	"""
+	This is an alternative to multiprocessing.Pool to avoid the limitations of the package (pickling stuff...)
+	
```

Comment:
Pool does not work because the function to "map" opens files to save results ? Or is it that the function to map is not defined in a cleanly importable way in a different module (something that multiprocessing.Pool wants, if I remember correctly) ?